src/styles/_variables/variables.scss

```diff
@@ -0,0 +1,15 @@
+$PrimaryFont: Roboto, sans-serif;
+$FontRegular: 400;
+$FontMedium: 500;
+$FontBold: 700;
+$ColorMainAccent: #060b35;
+$ColorSecondary: #616070;
+$ColorBlueAccent: #00acdc;
+$ColorDarkBlue: #34568b;
+$ColorCardBorder: #f3f3f3;
+$White: #fff;
+$CardWidth: 200px;
+$MediaPointPhone: 488px;
+$MediaPointTablet: 768px;
+$MediaPointDescktop: 1024px;
```

Comment:
There is a typo in the variable name `$MediaPointDescktop`. It should be `$MediaPointDesktop` to maintain consistency and avoid confusion.